src/Api/Model/Statistics/UserInfo.ts

```diff
@@ -0,0 +1,7 @@
+import { ItemsPerMonth } from './StatisticsResponse'
+
+export type UserInfo = {
+  readonly usersPerMonth: ItemsPerMonth[]
```

Comment:
see suggestions below

```suggestion
  readonly usersPerMonth: ItemsPerMonth
```